wordpress-mobile/WordPress-Android
# Jetpack focus: Connect jetpack powered overlay to badges

This PR connects Jetpack overlay with all **Jetpack powered** badges on the following

Fixes: #16977 

- Home
- Me
- App Settings
- Activity Detail Log
- Notifications Settings
- Reader Post details


| Bottom Sheet | Google Play | Jetpack app |
|---|---|---|
|![Screenshot_20220726_124506](https://user-images.githubusercontent.com/990349/180912546-399bd576-12e1-400c-8190-164757936506.png)| ![Screenshot_20220726_124826](https://user-images.githubusercontent.com/990349/180912637-8eab334a-c99b-4ae3-a43a-2a05e1fcb704.png)|![Screenshot_20220726_124553](https://user-images.githubusercontent.com/990349/180912679-af06a12a-b605-4ef0-94c6-e4ea6b0fc8ef.png)|

To test:

- Launch WP app
- Switch to home tab, find Jetpack powered badge, scroll down if necessary
- Tap on **Jetpack powered** badge, ensure it opens up bottom sheet as shown in  first image above
- Tap on **Get the new Jetpack app** button
- Ensure, it opens Google Play store if Jetpack app is not installed, otherwise Jetpack app itself

Try out tapping on **Jetpack powered** badge on the other places [as shown on this PR](https://github.com/wordpress-mobile/WordPress-Android/pull/16911)



## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit tests

3. What automated tests I added (or what prevented me from doing so)
Updated tests

PR submission checklist:
